DayNightQ10.Rmd

```diff
@@ -69,3 +69,30 @@ plot(x,y)
 cat("The Q10 Value is", calc_q10(x,y))
 ```
 
+## Ts Percentage
+```{r Ts Percentage, message = FALSE, warning=FALSE}
+main_results <- list()
+for(rhds in rh_dataset_names){
+  ds_data <- csr_dataset(rhds)
+  dsd <- ds_data$data
+  column_names <- names(dsd)
+  temperature_columns <- column_names[grep("^CSR_T(AIR|[0-9]+)", column_names)]
+  results <- data.frame(columns = temperature_columns, depth = NA, q10 = NA, n = NA)
+  for(tcol in temperature_columns){
+    temp <- dsd[[tcol]]
+    resp <- dsd$CSR_FLUX_CO2
+    results$q10[results$columns == tcol] = calc_q10(temp, resp)
+    results$n[results$columns == tcol] = length(temp[!is.na(temp) & !is.na(resp)])
+    depth <- gsub("^CSR_T", "", tcol)
+    if(!is.na(as.numeric(depth))){
```

Comment:
This could use a comment--explain what's happening here (why the `is.numeric` check).